crates/astria-core/src/primitive/v1/asset/denom.rs

```diff
@@ -257,6 +266,18 @@ impl TracePrefixed {
     pub fn push_trace_segment(&mut self, segment: PortAndChannel) {
         self.trace.push(segment);
     }
+
+    /// Calculates the length of the display formatted [`TracePrefixed`] without allocating a
+    /// String.
+    #[must_use]
+    #[allow(clippy::arithmetic_side_effects)] // allow: length should never overflow usize::MAX
```

Comment:
We are already on top of `1.81.0` - let's start using `#[expect(<LINT>, reason = "<reason>")]`.